nodeadm/cmd/nodeadm/init/init.go

```diff
@@ -28,6 +28,7 @@ const (
 func NewInitCommand() cli.Command {
 	init := initCmd{}
 	init.cmd = flaggy.NewSubcommand("init")
+	init.cmd.StringSlice(&init.daemons, "d", "daemon", "specify one or more of `containerd` and `kubelet`")
```

Comment:
I'm a bit hesitant to add this to the CLI, can we add a disclaimer that you probably don't want to do this? 

```suggestion
	init.cmd.StringSlice(&init.daemons, "d", "daemon", "specify one or more of `containerd` and `kubelet`. This is intended for testing and should not be used in a production environment.")
```

Reply:
> So this utility can be used for CI jobs outside of EKS

probing for more context, what specific usage do you have in mind?